.github/workflows/build.linux.workflow.yml

```diff
@@ -2,9 +2,9 @@ name: Build Linux
 
 on:
   push:
-    branches: [ master, experimental, experimental2 ]
+    branches: [ master, experimental, experimental2, feature/ci-windows ]
```

Comment:
For now, I'm keeping this because it gives you the ability to perform CI on your branch, but I guess I'll edit the commit post-merge and force push.

Reply:
Sure.